assets/src/stories-editor/store/reducer.js

```diff
@@ -159,4 +159,22 @@ export function blocks( state = {}, action ) {
 	}
 }
 
-export default combineReducers( { animations, currentPage, blocks } );
+export function copiedMarkup( state = {}, action ) {
+	switch ( action.type ) {
+		case 'SET_COPIED_MARKUP':
+			const { markup } = action;
+			return markup;
+
+		case 'CLEAR_COPIED_MARKUP':
+			if ( ! state.length ) {
```

Comment:
What is this guarding against? If someone sets `true` or `6` as the copied markup, then this means it can't be cleared, or?

Reply:
It's only supposed to save strings as markup, changed the logic a little bit.